agent-manager/.python-version

```diff
@@ -0,0 +1 @@
+3.13
```

Comment:
how does this related to `FROM registry.access.redhat.com/ubi9/python-312:latest` in the container file which seems to be python 312